templates/services_user_data.tpl

```diff
@@ -110,9 +110,10 @@ mkdir -p /etc/circleconfig/telegraf
 
 cat <<EOF > /etc/circleconfig/telegraf/telegraf.conf
 [global_tags]
-  service = "circleci"
-  type = "server"
+  service = "circleci-server"
+  app = "circleci"
   env = "${env}"
+  host = "circleci-svr-${env}"
```

Comment:
Why not `server` instead of `srv`?

Reply:
No explicit reason, mostly because I thought it might be too long.